priv/sample/src/sample_app_ping_pong.erl

```diff
@@ -0,0 +1,33 @@
+-module(sample_app_ping_pong).
```

Comment:
I think this can be avoided by telling rebar3 to no compile source dirs recursively (it'll just compile the top-level)
http://rebar3.org/docs/configuration/configuration/#enabledisable-recursive-compiling